src/api/firebase.js

```diff
@@ -72,10 +73,66 @@ export async function addItem(listId, { itemName, daysUntilNextPurchase }) {
 	});
 }
 
-export async function updateItem(listId, itemId, currentItemUpdates) {
+export async function updateItem(
+	listId,
+	itemId,
+	dateLastPurchased,
+	dateNextPurchased,
+	dateCreated,
+	currentItemUpdates,
+) {
```

Comment:
Tip for the future: Passing your function arguments as objects is better.

Right now, you're force to call this function like this

```js
updateItem("theListId", "theItemId", lastDate, nextDate, dateCreated, items)
```

But you're forced to remember the order of each argument, and now you have six of them 😱. Also, someone else reading this function wouldn't understand what each on of these really mean, they would have to read the function definition to understand what's going on.

If you use an object, you can do something like:

```js
updateItem({listId: "theListId", itemId: "theItemId", dateLastPurchased...})
```

**No need to change this!**

I usually tend to start using objects as soon

Reply:
Yeah, it makes sense to use an object since we need to make sure of the order of the argument. We got many errors due to the order.